skops/hub_utils/tests/test_hf_hub.py

```diff
@@ -271,6 +271,23 @@ def test_init(classifier_pickle, config_json):
         )
 
 
+def test_init_modelcard_creation(classifier_pickle, config_json):
+    # create a temp directory and delete it, we just need a unique name.
+    dir_path = tempfile.mkdtemp()
+    shutil.rmtree(dir_path)
+
+    version = metadata.version("scikit-learn")
+    init(
+        model=classifier_pickle,
+        requirements=[f'scikit-learn="{version}"'],
+        dst=dir_path,
+        task="tabular-classification",
+        data=iris.data,
+    )
+    _validate_folder(path=dir_path)
+    assert os.path.isfile(Path(dir_path) / "README.md")
```

Comment:
I wonder if we should simply add it under `test_init` to deduplicate (essentially everything before this step is same and this behavior will be a part of init anyway)? WDYT? @adrinjalali @jucamohedano

Reply:
That's true, this specific one can be in `test_init`.

Reply:
I agree. Made the change.